app/views/admin/stats/referrals/index.html.slim

```diff
@@ -0,0 +1,14 @@
+div.panel-body
+  div.adv-table
+    table.display.table.table-bordered.table-striped.dynamic-table id="dynamic-table"
+      tr
+        th Channel
+        th Earnings
+        th Publisher
+      tbody
+        - @result.each do |entry|
```

Comment:
Should we display a "No results found" message if there's nothing in the result array?